src/Sylius/Bundle/CoreBundle/Resources/config/app/messenger.yaml

```diff
@@ -13,6 +13,14 @@ framework:
         # Route your messages to the transports
         # 'App\Message\YourMessage': async
 
-        default_bus: sylius_default.bus
+        default_bus: sylius.command_bus
         buses:
-            sylius_default.bus: ~
+            sylius.command_bus: &command_bus
+                middleware:
+                    - 'validation'
+                    - 'doctrine_transaction'
+            sylius.event_bus: &event_bus
+                default_middleware: allow_no_handlers
+            sylius_default.bus: *command_bus
+            sylius_event.bus: *event_bus
```

Comment:
I'm not sure if these aliases work properly, did you check that? It looks like new buses are creating here